src/actions/apiCalls.ts

```diff
@@ -2,6 +2,7 @@ import { ApiEndpoint } from "../api";
 import ApiCallError from "../errors/ApiCallError";
 import { AsyncAction } from ".";
 
+Math.random();
```

Comment:
Why?

Reply:
Lol oops must have been trying something out

Reply:
oh i thought it might be some fancy anti-caching stuff or something lol